pylint_odoo/test_repo/broken_module/model_view_odoo2.xml

```diff
@@ -85,5 +85,25 @@
             </field>
         </record>
 
+        <!-- duplicate record field "name" in *2M field-->
+        <record id="view_model_form5" model="ir.ui.view">
+            <field name="name">view.model.form</field>
+            <field name="model">test.model</field>
+            <field name="arch" type="xml">
+                <form string="Test model6">
+                    <field name="picking_ids">
+                        <tree>
+                            <field name="name"/>
+                        </tree>
+                    </field>
+                    <field name="value_ids">
+                        <tree>
+                            <field name="name"/>
+                        </tree>
+                    </field>
+                </form>
+            </field>
+        </record>
+
```

Comment:
@hugho-ad 
FYI I ran the test and duplicated-xml-fields was emitted 7 times.
Maybe we need add the parent in the key in order to group all items, I mean:
```diff
- all_fields.setdefault(field_xml, []).append(field)
+ all_fields.setdefault((field_xml, field.getparent()), []).append(field)
```

And process even this multi key in all  other part of code where is used.